cms/templates/login_page.html

```diff
@@ -19,11 +19,11 @@ <h1 class="title title-1">${_("Sign In to {studio_name}").format(studio_name=set
     <article class="content-primary">
       <div class=""><!-- What class for a dive to wrap the form? -->
         <form method="post" action="${reverse('login')}">
-          %if show_login_error_message: 
+          %if login_error_message:
             <div id="login_error"
                  class="message message-status error is-shown"
                  style="display: block;">
-               ${_("Email or password is incorrect. Please ensure that you are a course staff in order to use Studio.")}
+               <p>${_(login_error_message)}</p>
```

Comment:
Using `gettext` or `_()` will not work on variables due to the use of static code analysis by `gettext`.

```suggestion
               <p>${_(login_error_message)}</p>
```